### DISCUSSION TITLE
rpm or rpm-rs

### DISCUSSION THREAD

**Discussion by drahnr** [rpm-rs/rpm#50] [General]
Currently we don't have access to `rpm-rs` as a crate name, but we have `rpm`. Imho `rpm` is more favorable. The `-rs` suffix is just redundant information after all, it's scoped to `rpm-rs` already.

**Comment by cmeister2** [comment]
The crate is also imported as `use rpm` too. IMHO `rpm` is the better name and we should just use it.

**Comment by dralley** [comment]
That's fair, my line of thinking is that

* `rpm` is the name of the official library written in C, shadowing the name isn't necessarily the worst thing but the clarification could be helpful
* You can't have both repos forked without renaming one of them
* If we wanted to have any binaries, we'd have to be careful about their naming to not have them confused with or shadow system utilities

**Comment by drahnr** [comment]
So far the focus was on providing a library, and for that, `use rpm::..` is easier. 

I think the addition of a library included being called rpm exists, especially due to the implicit rule of `Cargo.toml` and the presence of a main, but I don't think it mandates not moving on to the single most direct macht of a crate name we can get. I'd rather add a CI job to prevent having a `main.rs` without a binary section in the manifest, or a manifest section that creates an `rpm` binary.
